internal/cmd/controller/target/target_test.go

```diff
@@ -314,6 +329,70 @@ func TestProcessTemplateValues(t *testing.T) {
 		t.Fatal("join func was not right")
 	}
 
+	templatedValuesData, err := processTemplateValuesData(bundle.Helm.TemplateValues, values)
+	if err != nil {
+		t.Fatalf("error during label processing %v", err)
```

Comment:
:raised_eyebrow: Not sure why that error message is phrased this way (same for `processTemplateValues`)
```suggestion
		t.Fatalf("error during templated values processing %v", err)
```